src/main/java/com/coravy/hudson/plugins/github/GithubLinkAction.java

```diff
@@ -30,4 +35,24 @@ public String getUrlName() {
         return projectProperty.getProjectUrl().baseUrl();
     }
 
+    @SuppressWarnings({"rawtypes"})
+    @Extension
+    public static class Factory extends TransientActionFactory<Job> {
+        @Override
+        public Class<Job> type() {
+            return Job.class;
```

Comment:
maybe it should be `ParameterizedJobMixIn.ParameterizedJob.class` as already in property class descriptor?